internal/controllers/user_devices_controller.go

```diff
@@ -298,6 +299,28 @@ func (udc *UserDevicesController) dbDevicesToDisplay(ctx context.Context, device
 					sdStat.Address = &a
 				}
 			}
+
+			if cred := vnft.R.Claim; cred != nil {
+				var c map[string]interface{}
+				err = json.Unmarshal(cred.Credential, &c)
+				if err != nil {
+					return nil, err
+				}
+				subj, ok := c["credentialSubject"]
+				if ok {
+					v, ok := subj.(map[string]interface{})["vehicleIdentificationNumber"]
+					if ok {
+						credential = &CredentialData{
+							Vin:            v.(string),
+							ExpirationDate: cred.ExpirationDate,
+							IssuanceDate:   cred.IssuanceDate,
```

Comment:
Can we set `Valid` to `time.Now().Before(cred.ExpirationDate)`? One less if statement.